ynput/OpenPype
# Max: Adding custom modifiers back to the loaded objects

## Changelog Description
The custom parameters OpenpypeData doesn't show in the loaded container when it is being loaded through the loader.

## Additional info
<del>Loading Max Scene is a bit different from the rest.</del>
<del>It would be resolved in other ticket: https://github.com/ynput/OpenPype/pull/5406</del> 
<del>Should be merged before #5406Fixed the conflict</del>
Resolve #5406 
This ticket also affects #5434 
Related to "3dsMax-loading-an-asset-multiple-times"
## Testing notes:
1. Publish something in Max(any instance except render)
2. Load instances(e.g. model, camera etc.)
3. The OP Parameter should be embedded into the loaded container attribute 
![image](https://github.com/ynput/OpenPype/assets/64118225/dca27f89-abb4-43bd-a075-b28d2402d996)
4. Switch version 
5. Check the OP param and objects 
6. They should be updated
